theano/gpuarray/dnn.py

```diff
@@ -1135,6 +1135,11 @@ def dnn_conv3d(img, kerns, border_mode='valid', subsample=(1, 1, 1),
                                     desc_op.subsample)
     out_shp = assert_conv_shape(out_shp)
     out = gpu_alloc_empty(ctx_name, dtype=img.dtype)(*out_shp)
+    if config.dnn.conv.algo_bwd_filter != 'none':
+        warnings.warn('[new backend] cuDNN backward filter convolution computation for 3D convolutions '
```

Comment:
No, (1,1,1) does not cause problems.

PS: As recall, I test specific values with these inputs:
```
run_conv3d_bwd(inputs_shape, filters_shape, subsample, border_mode, conv_mode)
run_conv3d_bwd((8, 1, 20, 12, 15), (5, 1, 6, 6, 4), (1, 1, 1), 'full', 'conv')
```

I will check if (1,1,1) is ok even with different values for other parameters (input shape, filter shape, border mode, conv mode).